src/widgets/webAuthn/webAuthnDevicesWidget.tsx

```diff
@@ -151,3 +191,61 @@ export default createWidget<WebAuthnWidgetProps, WebAuthnDevicesProps>({
+import { ReactComponent as Dashlane } from '../../icons/webauthn/dashlane.svg'
+import { ReactComponent as Edge } from '../../icons/webauthn/edge.svg'
+import { ReactComponent as Enpass } from '../../icons/webauthn/enpass.svg'
+import { ReactComponent as GooglePasswordManager } from '../../icons/webauthn/google-password-manager.svg'
+import { ReactComponent as IDmelon } from '../../icons/webauthn/idmelon.svg'
+import { ReactComponent as KeePassXC } from '../../icons/webauthn/keepassxc.svg'
+import { ReactComponent as Keeper } from '../../icons/webauthn/keeper.svg'
+import { ReactComponent as NordPass } from '../../icons/webauthn/nordpass.svg'
+import { ReactComponent as ProtonPass } from '../../icons/webauthn/proton-pass.svg'
+import { ReactComponent as SamsungPass } from '../../icons/webauthn/samsung-pass.svg'
+import { ReactComponent as Thales } from '../../icons/webauthn/thales.svg'
+import { ReactComponent as WindowsHello } from '../../icons/webauthn/windows-hello.svg'
+
+const providerData: Map<string, [string, React.FunctionComponent<React.SVGAttributes<SVGElement>> | null]> = new Map([
+    ["ea9b8d66-4d01-1d21-3ce4-b6b48cb575d4", ["Google Password Manager", GooglePasswordManager]],
+    ["adce0002-35bc-c60a-648b-0b25f1f05503", ["Chrome on Mac", Chrome]],
+    ["08987058-cadc-4b81-b6e1-30de50dcbe96", ["Windows Hello", WindowsHello]],
+    ["9ddd1817-af5a-4672-a2b9-3e3dd95000a9", ["Windows Hello", WindowsHello]],
+    ["6028b017-b1d4-4c02-b4b3-afcdafc96bb2", ["Windows Hello", WindowsHello]],
+    ["dd4ec289-e01d-41c9-bb89-70fa845d4bf2", ["iCloud Keychain (Managed)", Apple]],
+    ["531126d6-e717-415c-9320-3d9aa6981239", ["Dashlane", Dashlane]],
+    ["bada5566-a7aa-401f-bd96-45619a55120d", ["1Password", OnePassword]],
+    ["b84e4048-15dc-4dd0-8640-f4f60813c8af", ["NordPass", NordPass]],
+    ["0ea242b4-43c4-4a1b-8b17-dd6d0b6baec6", ["Keeper", Keeper]],
+    ["f3809540-7f14-49c1-a8b3-8f813b225541", ["Enpass", Enpass]],
+    ["b5397666-4885-aa6b-cebf-e52262a439a2", ["Chromium Browser", null]],
+    ["771b48fd-d3d4-4f74-9232-fc157ab0507a", ["Edge on Mac", Edge]],
+    ["39a5647e-1853-446c-a1f6-a79bae9f5bc7", ["IDmelon", IDmelon]],
+    ["d548826e-79b4-db40-a3d8-11116f7e8349", ["Bitwarden", Bitwarden]],
+    ["fbfc3007-154e-4ecc-8c0b-6e020557d7bd", ["iCloud Keychain", Apple]],
+    ["53414d53-554e-4700-0000-000000000000", ["Samsung Pass", SamsungPass]],
+    ["66a0ccb3-bd6a-191f-ee06-e375c50b9846", ["Thales Bio iOS SDK", Thales]],
+    ["8836336a-f590-0921-301d-46427531eee6", ["Thales Bio Android SDK", Thales]],
+    ["cd69adb5-3c7a-deb9-3177-6800ea6cb72a", ["Thales PIN Android SDK", Thales]],
+    ["17290f1e-c212-34d0-1423-365d729f09d9", ["Thales PIN iOS SDK", Thales]],
+    ["50726f74-6f6e-5061-7373-50726f746f6e", ["Proton Pass", ProtonPass]],
+    ["fdb141b2-5d84-443e-8a35-4698c205a502", ["KeePassXC", KeePassXC]],
+])
+
+function getProviderData(aaguid: string): [string | null, React.FunctionComponent<React.SVGAttributes<SVGElement>>] {
```

Comment:
Are there any plans to support the [FIDO Alliance Metadata Service](https://fidoalliance.org/metadata/) ? We could avoid hard-coding this kind of information.